src/components/Pagination/JumpToPageForm.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import classnames from 'classnames';
+import Button from '../Button';
+
+export default function JumpToPageForm({ onPaginate, totalPages }) {
+  return (
+    <Formik
+      initialValues={{ page: '' }}
+      onSubmit={({ page }, { resetForm }) => {
+        onPaginate(page);
+        resetForm();
+      }}
+      validationSchema={Yup.object().shape({
+        page: Yup.number()
+          .required('Please enter a valid page number')
+          .min(1, 'Page number must be greater than 1')
+          .max(totalPages, 'Page number must be lower than ' + totalPages)
```

Comment:
Is ` + totalPages` supposed to be ` + (totalPages + 1)`?

Reply:
nice catch, fixed it:

![image](https://user-images.githubusercontent.com/1882507/51922935-ff41cb00-23b7-11e9-9fe3-3b36b105cf34.png)